geosys/bridge_api/definitions.py

```diff
@@ -211,3 +211,59 @@
     'name': 'SLOPE',
     'map_family': topology_map
 }
+
+ARCHIVE_MAP_PRODUCTS = [
+    NDVI,
+    EVI,
+    GNDVI,
+    CVI,
+    OM,
+    YVM,
+    YGM,
+    SAMZ,
+    COLOR_COMPOSITION,
+    ELEVATION
+]
+
+INSEASON_MAP_PRODUCTS = [
+    INSEASON_NDVI,
+    INSEASONFIELD_AVERAGE_NDVI,
+    INSEASONFIELD_AVERAGE_REVERSE_NDVI,
+    INSEASON_CVI,
+    INSEASON_EVI,
+    INSEASONPARTIAL_NDVI,
+    INSEASONPARTIAL_EVI,
+    INSEASON_GNDVI,
+    COLOR_COMPOSITION
+]
```

Comment:
@AndreLester @timlinux I'm not sure about the map products options. Coverage request with geometry parameter only returns map products included in `INSEASON_MAP_PRODUCTS`. Is this correct?

Reply:
I will follow this up with the client.